docs/index.asciidoc

```diff
@@ -117,6 +117,7 @@ filter {
     add_field => { user_firstname => "%{[user][0][firstname]}" } <5>
     add_field => { user_lastname => "%{[user][0][lastname]}" } <5>
     remove_field => ["server", "user"]
+    staging_directory => "/tmp/logstash/jdbc_static/import_data"
```

Comment:
should we include the loader_schedule example up here in this top example as well?

Reply:
Done.